contrib/scripts/sensitive_strings.py

```diff
@@ -109,7 +109,9 @@ def _is_binary_file(self, file_path: str, file_name_ext: str):
         ext = ext.lower()
         if ext == ".ipynb":
             is_binary_file = True
-        if self._is_img_ext(ext):
+        elif ext == ".h5":
```

Comment:
Is there no corresponding test update?

Reply:
Updated in commit https://github.com/sandialabs/OpenCSP/pull/157/commits/fd5c12d8c1cb7897c6429ec5c1338a3800c12577